federation/cmd/federation-apiserver/app/server.go

```diff
@@ -48,8 +48,8 @@ import (
 	"k8s.io/kubernetes/pkg/generated/openapi"
 	"k8s.io/kubernetes/pkg/kubeapiserver"
 	kubeapiserveradmission "k8s.io/kubernetes/pkg/kubeapiserver/admission"
-	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
-	kubeserver "k8s.io/kubernetes/pkg/kubeapiserver/server"
```

Comment:
> tengqm 7 hours ago
> okay. If it is a shared code, are we gonna move it over to apiserver project?

It is only shared for these two legacy servers.  No new server should ever serve insecurely, so we aren't going to move it into the generic apiserver.